src/configure/configurers/AksAzureResourceSelector.ts

```diff
@@ -0,0 +1,12 @@
+import { GenericResource } from "azure-arm-resource/lib/resource/models";
+import { TargetResourceType } from "../model/models";
+import { IAzureResourceSelector } from "./IAzureResourceSelector";
+
+export class AksAzureResourceSelector implements IAzureResourceSelector {
+     async getAzureResource(inputs): Promise<GenericResource> {
+          return { type: TargetResourceType.AKS };
+
```

Comment:
Also, we are not checking whether a pipeline is already configured for a AKS resource (namespace?). Is that because we can always create a new pipeline targeting a new namespace? If yes, it sounds okay. But then do we never show browse experience for AKS ? @anuragc617 your thoughts

Reply:
BTW I am digressing here so this is a low priority comment.

Reply:
@bishal-pdMSFT we don't show browse experience for AKS because we can always create a new pipeline even in the same namespace/ new namespace.